atlas/logback/logback.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<configuration>
+
+    <appender name="standardOut" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <Pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</Pattern>
+        </encoder>
+    </appender>
+
+    <logger name="org.mapfish" level="WARN" />
+    <logger name="ch.thus.camel.retry" level="WARN" />
```

Comment:
It actually is:
https://github.com/georchestra/georchestra/blob/Atlas/atlas/src/docker/Dockerfile#L8